romanisim/wcs.py

```diff
@@ -191,9 +192,19 @@ def _radec(self, x, y, color=None):
         if np.ndim(x) == np.ndim(y) == 0:
             return r[0], d[0]
         else:
-            assert np.ndim(x) == np.ndim(y)
-            assert x.shape == y.shape
-            return r, d
+            # assert np.ndim(x) == np.ndim(y)
+            # assert x.shape == y.shape
+            if ((np.ndim(x) == np.ndim(y)) and (x.shape == y.shape)):
+                return r, d
+            else:
+                if (np.ndim(x) != np.ndim(y)):
+                    raise Exception("np.ndim(x) != np.ndim(y) => " + str(
+                        np.ndim(x)) + " != " + str(np.ndim(y)))
+                elif (x.shape != y.shape):
+                    raise Exception("x.shape != y.shape => " + str(
+                        x.shape) + " != " + str(y.shape))
+                else:
+                    raise Exception("Invalid x & y.")
```

Comment:
I would suggest you use a specific exception:
```suggestion
                    raise ValueError(f"np.ndim(x) != np.ndim(y) => {np.ndim(x)} != {np.ndim(y)}")
                elif (x.shape != y.shape):
                    raise ValueError(f"x.shape != y.shape => {x.shape} != {y.shape}")
                else:
                    raise ValueError("Invalid x & y.")
```

Reply:
Changed in both blocks.